### PR TITLE
WRQ-23779: Added missing ui/ss tests

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added missing ui/ss tests

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added missing ui/ss tests

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-23779

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: tests/screenshot/apps/components/ContextualPopupDecorator.js
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/ContextualPopupDecorator.js:1]: Please add some tests with ContextualPopupButtonNoArrow for Arabic too. You could also separate the sets of tests with comments for ContextualPopupButton and ContextualPopupButtonNoArrow.

File: tests/ui/apps/PageViews/PageViews-View.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import {BasicArranger} from '../../../../internal/Panels';
+import {PageViews} from '../../../../PageViews';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import spotlight from '@enact/spotlight';
+import Item from "../../../../Item";
```
Comment (stanca-pop-lgp) [inline tests/ui/apps/PageViews/PageViews-View.js:5]: Please move this import after internal/Panels.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

File: tests/ui/apps/Checkbox/Checkbox-View.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Checkbox from "../../../../Checkbox";
```
Comment (stanca-pop-lgp) [inline tests/ui/apps/Checkbox/Checkbox-View.js:1]:
```suggestion
import Checkbox from '../../../../Checkbox';
```

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/ContextualPopupDecorator.js [hunk 3]
```diff
@@ -35,14 +42,38 @@ const ContextualPopupDecoratorTests = [
 		<ContextualPopupButton direction="right top" />,
 		<ContextualPopupButton direction="right bottom" />,
 		// *************************************************************
+		// ContextualPopupButton with no arrow
+		// *************************************************************
+		<ContextualPopupButtonNoArrow offset="none" />,
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/ContextualPopupDecorator.js:47]: 'offset' prop test is duplicated by line 74~76. I think it need to remove 'offset' prop in below 'NoArrow' test.
```suggestion
		<ContextualPopupButtonNoArrow />,
		
```

**Review by dongsuwon — CHANGES_REQUESTED** [review]

File: tests/screenshot/apps/components/ContextualPopupDecorator.js (hunk 3 above)
Comment (paul-beldean-lgp) [reply tests/screenshot/apps/components/ContextualPopupDecorator.js:47]: The tests from 74-76 are for the RTL locale and I think it's useful to have them. Also the 'offset' prop is required for the test.

Comment (dongsuwon) [reply tests/screenshot/apps/components/ContextualPopupDecorator.js:47]: You are right. I missed it.

**Review by dongsuwon — APPROVED** [review]

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**